lib/workers/repository/index.js

```diff
@@ -15,13 +16,13 @@ module.exports = {
 // istanbul ignore next
 async function renovateRepository(repoConfig) {
   let config = { ...repoConfig };
-  logger.setMeta({ repository: config.repository });
-  logger.info('Renovating repository');
-  logger.trace({ config });
+  const repositoryLogger = logger.child({ repository: config.repository });
```

Comment:
Do we need a child logger here?

Reply:
Nope. This is a result of me trying to rebase this branch onto master. I want to get this merged ASAP to prevent errors from this happening again